src/main/flight/mixer.c

```diff
@@ -540,6 +540,15 @@ void FAST_CODE mixTable()
 
         motorValueWhenStopped = getReversibleMotorsThrottleDeadband();
         mixerThrottleCommand = constrain(rcCommand[THROTTLE], throttleRangeMin, throttleRangeMax);
+
+        if(feature(FEATURE_REVERSIBLE_MOTORS) && reversibleMotorsThrottleState == MOTOR_DIRECTION_BACKWARD) {
```

Comment:
If this applies only for DSHOT, then `if DSHOT` should be added to this condition. Code works fine for non-DSHOT protocols and making that change would break that

Reply:
Thanks a lot for your reply! I have added the check for DSHOT protocol on run time.

Reply:
Thanks a lot for your reply and feedback! I have added the check at compile and at runtime. Is that ok for your?